community/modules/network/private-service-access/variables.tf

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "network_id" {
```

Comment:
nit level: we have a slight preference for network_self_link over network_id, but I think both have similar properties (esp. both contain the project ID).

Reply:
They do, I wasn't sure if the module took the self link, since they explicitly say id, so I chose to go by their docs instead.
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_global_address